frontend/src/scenes/session-recordings/templates/sessionRecordingTemplatesLogic.tsx

```diff
@@ -131,10 +135,14 @@ export const sessionReplayTemplatesLogic = kea<sessionReplayTemplatesLogicType>(
         ],
         editableVariables: [(s) => [s.variables], (variables) => variables.filter((v) => !v.noTouch)],
     }),
-    listeners(({ values }) => ({
+    listeners(({ values, props }) => ({
         navigate: () => {
+            posthog.capture('session replay template used', {
+                template: props.template.key,
+                category: props.category,
+            })
```

Comment:
we _could_ make every card have a button and then carry on using autocapture e.g. https://us.posthog.com/project/2/insights/uu9nALhj

as things stand you don't know if a card click will navigate you or ask you for variables. 

so you could always have `(apply|use|something) template` as a button on every card

2 clicks but consistent behaviour.

really no strong opinion though

Reply:
I generally prefer custom events and only use autocapture when I need to because I want to dig into historical things. So I will keep the custom event, but will play around with the double-click-no-variables thing too. I agree it might be jarring to click one of those and suddenly be redirected away.